src/v/config/node_config.h

```diff
@@ -146,6 +149,14 @@ struct node_config final : public config_store {
         return data_directory().path / "cloud_storage_inventory";
     }
 
+    std::filesystem::path datalake_staging_path() const {
+        if (datalake_staging_directory().has_value()) {
+            return std::string(datalake_staging_directory().value());
```

Comment:
Hm, I'm wondering if we should even provide the option to set this. Since we're already thinking about a policy for space management to interact with this, would it make our lives easier to assume/enforce that this is always in the data directory?

Reply:
yeh, good idea. see https://github.com/redpanda-data/redpanda/pull/25052